### ISSUE TITLE
BUG: Initial example plots tab - disappearing plots when increasing Prevalence

### ISSUE THREAD

**Issue by menriquez-IDM** [InstituteforDiseaseModeling/LeakyVaccine#32]
## Repro:
- Open https://leakyvaccine-acc.bmgf.io/
- Go to "Initial example plots tab"
- Without moving any other parameter,  set Prevalence  parameter to 0.1, see Figure #2
- now  to 0.2,  see Figure #2
- now to 0.3,  see Figure #2, and 3
- now to 0.4 ... see Figure #2, and 3 
## EXPECTED
- Figure continues rendering the Plot which adjusts the Y axes and X axes to fit the resulting values
- Or if the value selected is invalid, it displays a warning
## ACTUAL
- As the value starts increasing (.1, .2, .3 etc) the charts start moving up, to the point that they get completely out of frame, and it eventually 'disappear'  

![image](https://user-images.githubusercontent.com/74386190/141365155-81d362ad-58ce-4792-8b11-80550c8b1306.png)

**Comment by JHerbeck-IDM** [comment]
Good catch. One partial fix is to reduce the max of "prev" to be 0.3 (which is 30% prevalence of HIV viremia, which is unrealistic currently).

**Comment by JHerbeck-IDM** [comment]
Another fix would be to make the y-axis dynamic based on model output. I can see if I can do that.
